### DISCUSSION TITLE
utils.pyのmove_base_goal関数について

### DISCUSSION THREAD

**Discussion by arata** [hsr-project/general#2] [Q&A]
自作のプログラムでutils.pyをインポートし、move_base_goalを使用したとき
ロボットが動かず、move_arm_initを前に入れると動くことを確認しました。[(参考プログラム)](https://github.com/devrt/wrs2020-robocup-challenge/blob/main/wrs_challenge/scripts/move.py)
しかし、notebookの[3_navigation.ipynb](https://github.com/hsr-project/notebooks/blob/master/3_navigation.ipynb)や直接rostopic pubコマンドでアクションを投げる方法、
インタラクティブモードのpythonではmove_arm_initがなくても動作しました。
この２つの挙動の違いはバグでしょうか、なにか原因がありましたら教えていただけませんか。

**Comment by yosuke** [comment]
返信がかなり遅くなってすみませんでした。

参考プログラムの場合、移動する前に
```python
# 視線を少し下げる
utils.move_head_tilt(-0.4)
```
というコードが入っています。
障害物回避をするために視線を下げているのですが、視線を下げるとHSRのハンドが障害物回避に利用する頭部3Dカメラに映り込んで障害物として認識されてしまい、initポーズにしてハンドをうまく隠さないと移動できなくなっているのではないかと思います。

notebookの方は視線を下げるコマンドが入っていません。
